docs/source/qstartguide/index.rst

```diff
@@ -90,20 +90,24 @@ The output of the command will show the expid of the experiment and generate the
       - User scripts and  project code. (empty)
 
 
-Then, execute ``autosubmit create <expid> -np`` and Autosubmit will generate the workflow graph.
+Then, execute ``autosubmit create <EXPID> -np`` and Autosubmit will generate the workflow graph.
+
+.. runcmd:: autosubmit create a000 -o png --hide
 
 Run and monitoring
 ==================
 
- To run an experiment, use ```autosubmit run <expid>```. Autosubmit runs experiments performing the following operations:
+ To run an experiment, use ```autosubmit run <EXPID>```. Autosubmit runs experiments performing the following operations:
 
  - First, it **checks the experiment configuration**. If it is wrong, it won't proceed further.
  - Second, it **runs the experiment while retrieving all logs** from completed or failed tasks as they run.
  - Third, it manages all the **workflow steps by following the dependencies defined by the user** until all jobs are in COMPLETED or FAILED status. There can be jobs left in **WAITING** status if their dependencies are in **FAILED** status.
 
-While the experiment is running, it can be visualized via ``autosubmit monitor <expid>``.
+While the experiment is running, it can be visualized via ``autosubmit monitor <EXPID>``.
+
+.. runcmd:: find /home/docs/autosubmit/a000/plot/ -iname "*a000_*.png" -true -exec mv -- {} /home/docs/checkouts/readthedocs.org/user_builds/autosubmit/checkouts/2366/docs/source/qstartguide/dummy.pdf \;
```

Comment:
Hmmm, maybe we can think in a one to make this simpler & portable. I cannot run this on my computer as it will fail.